pyproject.toml

```diff
@@ -10,23 +10,24 @@ find = {}
 [project]
 name = "ocsmesh"
 authors = [
-    {name = "Jaime R Calzada", email = "jreniel@gmail.com"},
-    {name = "Soroosh Mani", email = "soroosh.mani@noaa.gov"}
+    {name = "Felicio Cassalho", email = "felicio.cassalho@noaa.gov"},
+    {name = "Soroosh Mani", email = "soroosh.mani@noaa.gov"},
+    {name = "Jaime R Calzada", email = "jreniel@gmail.com"}
 ]
 maintainers = [
-    {name = "Soroosh Mani", email = "soroosh.mani@noaa.gov"}
+    {name = "Felicio Cassalho", email = "felicio.cassalho@noaa.gov"}
 ]
-description = "Package to generate computational unstructured meshes from planetary modeling."
+description = "Package to generate computational unstructured meshes for ocean simulation."
 license = {file = "LICENSE"}
 readme = "README.md"
-requires-python = '>=3.9, <3.14' # 3.8 -> scipy
+requires-python = '>=3.9, <3.13' # 3.8 -> scipy
```

Comment:
Do we need to put the version boundary here? If it's because of triangle, we should probably not put this limitation, since triangle is an optional package